File: MasterAndroid/Udemy-AndroidS/05.OCAReverzed/app/src/main/java/com/magicbytes/main_menu/MainActivity.java
package com.magicbytes.main_menu;

import android.os.Bundle;
import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import com.example.backend_app_info.AppInfoCompleter;
import com.magicbytes.main_menu.R;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle bundle) {
        super.onCreate(bundle);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main_menu);
        if (bundle == null) {
            new AppInfoCompleter(this).complete();
        }
    }
}

File: MasterAndroid/Books/HowToBuild/01.ActivityCallbacks/app/src/main/java/com/example/activitycallbacks/MainActivity.kt
package com.example.activitycallbacks

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Log.d(TAG, "onCreate")
    }
    companion object {
        private const val TAG = "MainActivity"
    }
}